Source/WebKit/UIProcess/API/Swift/WebView.swift

```diff
@@ -140,4 +157,67 @@ extension WebViewRepresentable: NSViewRepresentable {
 }
 #endif
 
+@MainActor
+private final class Coordinator {
```

Comment:
Since we’re declaring this top level, is there a more descriptive name it could get?